README.md

```diff
@@ -42,6 +44,22 @@ Add DevExtreme to an Angular application (follow [this](https://github.com/DevEx
 > devextreme add devextreme-angular
 ```
 
+### <a name="add-devextreme-react"></a>React 
+
+Add DevExtreme to an React application:
+
+#### <a name="add-devextreme-react-no-global"></a>Without global package usage
```

Comment:
I suggest 
"Package installed locally" and "Package installed globally" 
and change it in all other places

Reply:
Maybe "Package installed globally" and "Without installing the package globally"
@nightskylark What do you think?

Reply:
It's not about working with locally installed packages. It's about runtime packages installation with `npx`.

Reply:
@dxvladislavvolkov It' s OK

Reply:
OK, then, let's it leave as is.